cli/compose/convert/service.go

```diff
@@ -123,6 +131,7 @@ func convertService(
 				TTY:             service.Tty,
 				OpenStdin:       service.StdinOpen,
 				Secrets:         secrets,
+				Privileges:      &privileges,
```

Comment:
I think it's better to leave `Privileges` as nil if there is nothing related to it specified in the compose file.